website/pages/en/_meta.js

```diff
@@ -27,11 +27,17 @@ export default {
     type: 'separator',
   },
   '###2': {
+    type: 'heading',
+    title: 'Substreams',
+  },
+  substreams: {
+    type: 'children'
```

Comment:
added new prop to theme to not render `folder` element and render directly his files (sidebar links for substreams docs in our case)